api/app/signals/apps/zds/workflow.py

```diff
@@ -0,0 +1,33 @@
+from django.conf import settings
```

Comment:
we have HEROPEND as new status
add comment to workflow.py, these should be kept in sync